wandb/sdk/launch/docker.py

```diff
@@ -170,9 +170,17 @@ def build_docker_image_if_needed(
     if launch_project.name:
         name_line = "ENV WANDB_NAME={wandb_name}\n"
     dockerfile = (
-        "FROM {imagename}\n" "{copy_code_line}" "{requirements_line}" "{name_line}"
+        "FROM {imagename}\n"
+        "RUN mkdir -p {homedir}/.cache\n"
```

Comment:
this line looks like a duplicate of the line below?

Reply:
Ah, nice catch